src/styles/utils/mixin.scss

```diff
@@ -0,0 +1,23 @@
+@mixin number-day {
+  @for $cell from 1 through 31 {
+    & :nth-child(#{$cell})::before {
+      content: '#{$cell}';
+    }
+  }
+}
+
+@mixin weak-day-lentgh {
+  @each $week-day, $index in $start-day {
+    &--start-day-#{$week-day} :nth-child(1) {
+      margin-left: $cellSize * $index + $index;
```

Comment:
Ensure that the `$cellSize` and `$index` variables include units. If they are unitless, the multiplication will not result in a valid `margin-left` value. For example, if `$cellSize` is `10` and `$index` is `1`, the result should be `10px * 1`, not `10 * 1`.